subsys/bluetooth/Kconfig

```diff
@@ -83,6 +83,34 @@ config BT_BROADCASTER
 
 endmenu
 
+config BT_EXT_ADV
+	bool "Extended Advertising and Scanning support [EXPERIMENTAL]"
+	help
+	  Select this to enable Extended Advertising API support.
+	  This enables support for advertising with multiple advertising sets,
+	  extended advertising data, and advertising in long range mode.
+	  It enables support for receiving and scanning extended advertising
```

Comment:
It enables support for receiving extended advertising data as a scanner, including support for advertising data over the coded PHY.